resources/manifest_schema.json

```diff
@@ -0,0 +1,156 @@
+{
+  "anyOf": [
+    {
+      "type": "array",
+      "minItems": 1,
+      "items": {
+        "$ref": "#/project"
+      }
+    },
+    {
+      "$ref": "#/project"
+    }
+  ],
```

Comment:
`"validation_error": {
"sub_errors": [
  "{'titdle': 'test_project0', 'url': ['www.github.com/knex/'], 'analysis': [{'url': 'here-you-find-the-analysis.com', 'description': 'thats what this analysis is about'}], 'authors': [{'name': 'test_owner0', 'email': 'email@address.com'}], 'team': {'team_name': 'teamname'}, 'status': 'DONE', 'description': 'can this be a multiline description?', 'date_creation': '1970-01-01', 'date_last_updated': '2017-05-11', 'tags': ['tag', 'bio'], 'future_work': 'this is something related to this work', 'related_projects': [{'id': 0, 'name': 'name of the listed project', 'author': 'author', 'url': 'www.my.url'}]} is not of type 'array'",
  "Additional properties are not allowed ('titdle' was unexpected)",
  "'title' is a required property"
],

`
`
"upper_errors": [
  "{'titdle': 'test_project0', 'url': ['www.github.com/knex/'], 'analysis': [{'url': 'here-you-find-the-analysis.com', 'description': 'thats what this analysis is about'}], 'authors': [{'name': 'test_owner0', 'email': 'email@address.com'}], 'team': {'team_name': 'teamname'}, 'status': 'DONE', 'description': 'can this be a multiline description?', 'date_creation': '1970-01-01', 'date_last_updated': '2017-05-11', 'tags': ['tag', 'bio'], 'future_work': 'this is something related to this work', 'related_projects': [{'id': 0, 'name': 'name of the listed project', 'author': 'author', 'url': 'www.my.url'}]} is not valid under any of the given schemas"
],
}`